src/Data/OpenUnion.hs

```diff
@@ -23,6 +23,7 @@ module Data.OpenUnion
   , decomp
   , weaken
   , extract
+  , intro
```

Comment:
I'm strongly of the opinion that this entire module should be `Internal` --- real users have no use for `decomp`, `weaken`, `extract` or really anything other than `Member`.